mmv1/products/firebasehosting/api.yaml

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: FirebaseHosting
+display_name: Firebase Hosting
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://firebasehosting.googleapis.com/v1beta1/
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+objects:
+  - !ruby/object:Api::Resource
+    name: 'Site'
+    min_version: beta
+    base_url: projects/{{project}}/sites/{{site}}
+    self_link: "{{name}}"
+    create_url: projects/{{project}}/sites?siteId={{site}}
+    update_verb: :PATCH
+    update_mask: true
+    description: 'A `Site` represents a Firebase Hosting site.'
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://firebase.google.com/'
+      api: 'https://firebase.google.com/docs/reference/hosting/rest/v1beta1/projects.sites'
+    parameters:
+      - !ruby/object:Api::Type::String
+        name: 'site'
+        description: |
+          Required. Immutable. A globally unique identifier for the Hosting site. This identifier is
+          used to construct the Firebase-provisioned subdomains for the site, so it must also be a valid
+          domain name label.
+        input: true
+        url_param_only: true
+    properties:
+      - !ruby/object:Api::Type::String
+        name: name
+        description: |
+          Output only. The fully-qualified resource name of the Hosting site, in the
+          format: projects/PROJECT_IDENTIFIER/sites/SITE_ID PROJECT_IDENTIFIER: the
+          Firebase project's
+          [`ProjectNumber`](https://firebase.google.com/docs/projects/api/reference/rest/v1beta1/projects#FirebaseProject.FIELDS.project_number) ***(recommended)*** or its
+          [`ProjectId`](https://firebase.google.com/docs/projects/api/reference/rest/v1beta1/projects#FirebaseProject.FIELDS.project_id).
+          Learn more about using project identifiers in Google's
+          [AIP 2510 standard](https://google.aip.dev/cloud/2510).
+        output: true
+      - !ruby/object:Api::Type::String
+        name: appId
+        description: |
+          Optional. The [ID of a Web App](https://firebase.google.com/docs/projects/api/reference/rest/v1beta1/projects.webApps#WebApp.FIELDS.app_id)
+          associated with the Hosting site.
```

Comment:
not critical, but consider adding:

- !ruby/object:Api::Type::String
        name: defaultUrl
        output: true
        description: |
          The default URL for the site in the form of `https://{name}.web.app`.

it is useful output information for developers.